### DISCUSSION TITLE
Custom sort comparer for @ngrx/signals/entities

### DISCUSSION THREAD

**Discussion by tim-wie** [ngrx/platform#4483] [Q&A]
Hello all,

is there any plan, to support a custom sort comparer for the signal based entities? Or is there already a solution, that i haven't  found?
I mean, like the custom sortComparer in the `@ngrx/entities` **EntityAdapter**.

Or should I just go for the `withComputed` feature, and create a computed signal, that sorts the entities like the BooksStore example, with the `@ngrx/signals` **SignalStore**?

**Comment by Ross-Blakeney** [comment]
~~Wondering about this as well. I went the route of setting up a `withComputed` and it worked for all scenarios aside from when removing entities via `removeEntity(...)` or `removeEntities(...)`. I'm not sure why the computed signal is not updated after removal in this manner.~~

~~So it leads me to think that this is not the intended method of exposing sorted entities.~~

Well, I was not bound to the computed value, so ignore the above.

Using the computed approach DOES seem to be the way to go.
